fix for:
src/parse-simulation.c:61: error: redefinition of typedef 'parse_state'
author: Patryk Belzak (Wroclaw University)
--- src/parse-simulation.c.orig	2020-08-02 07:59:31.000000000 +0200
+++ src/parse-simulation.c	2021-02-24 11:12:02.385501723 +0100
@@ -28,7 +28,7 @@
 #include "lssi.h"
 #include "nullable.h"
 
-typedef struct parse_state
+struct parse_state
 {
   // Path of state-items the parser has traversed.
   struct si_chunk
@@ -58,7 +58,7 @@
   // Causes chunk contents to be freed when the reference count is
   // one. Used when only the chunk metadata will be needed.
   bool free_contents_early;
-} parse_state;
+} ;
 
 
 static void
